jetbrains-core/src/software/aws/toolkits/jetbrains/services/codemodernizer/model/CodeModernizerSessionContext.kt

```diff
@@ -145,6 +151,8 @@ data class CodeModernizerSessionContext(
                         }
                         it.putNextEntry(paddedPathString, file.inputStream)
                     }
+                    // 4) Build Log
+                    it.putNextEntry(Path(BUILD_LOG_PATH).toString(), buildLogBuilder.toString().byteInputStream())
```

Comment:
Just to confirm: this will put the TXT file in root dir of the upload ZIP right?

Reply:
So that I can match for VS Code